### DISCUSSION TITLE
There is an error in the API Migration tutorial

### DISCUSSION THREAD

**Discussion by ghost** [SeaQL/sea-orm#1315] [Show and tell]
https://www.sea-ql.org/sea-orm-tutorial/ch01-03-migration-api.html
```
+   let schema_manager = SchemaManager::new(db);
+   Migrator::refresh(db).await?;
```

it should be like this
```
+    let schema_manager = SchemaManager::new(&db);
+    migrator::Migrator::refresh(&db).await?;
```

**Comment by tyt2y3** [comment]
Thanks for reporting. A small PR against https://github.com/SeaQL/sea-orm-tutorial/ will very much be appreciated!

**Comment by billy1624** [comment]
Hey @JediMaster25, nice catch!
